packages/core/overlays/core-overlays-browser-internal/src/banners/banners_service.test.mocks.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { type IOverlayBannersStart } from './banners_service';
+
+// internal duplicate of public mock for `createStartContractMock`
```

Comment:
The mock is used for unit tests. I created a duplicate of `createStartContract` to KIS.